ggml.h

```diff
@@ -370,6 +370,7 @@ extern "C" {
         GGML_TYPE_I64     = 27,
         GGML_TYPE_F64     = 28,
         GGML_TYPE_IQ1_M   = 29,
+        GGML_TYPE_Q4_0_AARCH64 = 30,
```

Comment:
We fixed it in the latest commit. We split our single consolidated interleaved layout type Q4_0_AARC64 type into three separate generic types: Q4_0_4_4, Q4_0_4_8, and Q4_0_8_8 (depending on the number of rows interleaved and block length per row), so that others who intend to use interleaved data packing can use these types. This is no longer restricted to specific hardware.